src/test/kotlin/com/jetbrains/snakecharm/fixtures/SnakemakeResolveTestCase.kt

```diff
@@ -33,35 +24,17 @@ abstract class SnakemakeResolveTestCase : SnakemakeTestCase() {
             }
             return element as T
         }
-
-        protected fun assertIsBuiltin(element: PsiElement?) {
-            assertNotNull(element)
-            assertTrue(PyBuiltinCache.getInstance(element).isBuiltin(element))
-        }
-    }
-
-    protected abstract fun doResolve(): PsiElement?
-
-    protected fun <T : PsiElement> assertResolvesTo(langLevel: LanguageLevel,
-                                                    aClass: Class<T>,
-                                                    name: String): T {
-        val result = Ref<T>()
-
-        runWithLanguageLevel(
-                langLevel,
-                Runnable { result.set(assertResolvesTo(aClass, name, null)) }
-        )
-
-        return result.get()
     }
 
-    protected fun <T : PsiElement> assertResolvesTo(aClass: Class<T>, name: String): T {
-        return assertResolvesTo(aClass, name, null)
+    private fun doResolve(): PsiElement? {
+        val ref = fixture?.getReferenceAtCaretPosition("resolve/" +
```

Comment:
Here is ok to expect that `fixture != null` because it is true if test setup was called. Also let's make 2 cases different: unresolved reference case (a bug in resolve implementation), no reference at caret (a bug in test data). I've already fixed that, no action required.